OpenRA.Game/Traits/Player/FrozenActorLayer.cs

```diff
@@ -47,6 +47,7 @@ public class FrozenActor
 		readonly IHealth health;
 
 		public bool Visible = true;
+		public bool Hidden = false;
```

Comment:
Can we get an explanation (comment) here for what this does? One would assume this is always just `!Visible`.